pkg/didcomm/transport/http/inbound.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package http
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+// inboundCommHTTPOpts struct for path and handler represented
+// as an inbound http option
+type inboundCommHTTPOpts struct {
+	Path        string
+	HandlerFunc http.HandlerFunc
+}
+
+// InboundHTTPSetting is an inbound HTTP transport options setting function
+type InboundHTTPSetting func(opts *inboundCommHTTPOpts)
+
+// InboundDIDCommHandler struct to pass details for request handlers
+type InboundDIDCommHandler struct {
+	handlerOpts *inboundCommHTTPOpts
+}
+
+// WithInboundSetting option is for creating an Inbounnd HTTP transport using an http.Handler
+func WithInboundSetting(inboundPath string, handlerFunc http.HandlerFunc) InboundHTTPSetting {
```

Comment:
I don't think we typically use the term "setting"

Reply:
removed function